.github/CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 <h3>New features since last release</h3>
 
+- The TensorFlow interface supports calculating the Hessian. The Hessian
+  currently only works with `diff_method='parameter-shift`.
```

Comment:
I asked a similar question for the torch PR, but here isn't there potential to support a backprop approach since we have `default.qubit.tf`?

Reply:
Yep, right - this line should be changed.